app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,56 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .dev-tag{
+  color: black;
+  width: 30px;
+  background-color: #A80874;
```

Comment:
Color literals like `#A80874` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A80874` should be written as `#a80874`